src_python/v0.1.1/config/env.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from pathlib import Path
+
+
+def read_env_var(name: str, default=None):
+    return os.environ.get(name, default)
+
+# I can't decipher how STL bash detects steam deck, so I'm using this instead
```

Comment:
It checks if the file `/etc/os-release` exists and if `steamdeck` is present in that file. Some distributions (i.e. Fedora) use `lsb-release`, but Arch-based distros including SteamOS will always use `os-release` (at least afaik right now), so we use this.

Not sure if there's a performant `grep` alternative in Python aside from checking a file manually. This is part of what I meant about Bash being mostly suitable for STL :-)